openstack/neutron/templates/statefulset-network-agent-apod.yaml

```diff
@@ -56,6 +56,22 @@ spec:
         multus: bond1
         failure-domain.beta.kubernetes.io/zone: {{ $az_long }}
         kubernetes.cloud.sap/apod: {{ $apod }}
+      initContainers:
+        - name: init
+          image: {{$.Values.global.dockerHubMirror}}/library/{{$.Values.imageNameNetworkAgentDHCPInit | required "Please set neutron.imageNameNetworkAgentDHCPInit or similar"}}:{{$.Values.imageVersionNetworkAgentDHCPInit | required "Please set neutron.imageVersionNetworkAgentDHCPInit or similar"}}
+          securityContext:
+            privileged: true
+          command:
+            - sh
+            - -c
+          args:
+            - |-
+              set -xe
+              chroot /host modprobe ebtable_nat
+              chroot /host lsmod | grep ebtable
```

Comment:
I think you can remove this additional check for the module because `modprobe` will return `1` exit code anyway. But it's up to you

Reply:
Good point, I agree. Will remove the extra check.